src/main/com/fulcrologic/fulcro_i18n/i18n.cljc

```diff
@@ -77,7 +77,7 @@
   "Ensure that the given locale is loaded. Is a no-op if there are translations in app state for the given locale
   which is a keyword like :es-MX."
   [app locale]
-  (let [state-map @(:com.fulcrologic.fulcro.application/state-atom app)]
+  (let [state-map @(::app/state-atom app)]
```

Comment:
shorter

Reply:
`(let [state-map (app/current-state app)]` is the same length, but uses Fulcros "official" api

Reply:
ah, so that returns a state-map and not a state, which was the problem before.